extension/extension_test.go

```diff
@@ -113,6 +113,11 @@ func TestBuilder(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotNil(t, e)
 
+	// Check that the extension has access to the resource attributes.
+	nop, ok := e.(nopExtension)
+	assert.True(t, ok)
+	assert.Len(t, nop.CreateSettings.ResourceAttrs, 0)
```

Comment:
Should we also have a test where `ResourceAttrs` is not empty?

Reply:
In this case we always expect it to be empty here because it's not being instantiated by a service

Reply:
we already have extensive tests for the service and getting the resources from there... maybe i'll edit one of them to ensure the expected resources exist in the telemetry settings.